registry/storage/driver/azure/azure.go

```diff
@@ -86,7 +86,8 @@ func New(accountName, accountKey, container, realm string) (*Driver, error) {
 	blobClient := api.GetBlobService()
 
 	// Create registry container
-	if _, err = blobClient.CreateContainerIfNotExists(container, azure.ContainerAccessTypePrivate); err != nil {
```

Comment:
It's the default. The header is optional when create container; without the header set, the container will be private. The new SDK doesn't provide the option to set the header. To change the permission, need to call Container.SetPermissions.